entrypoint/selinux/selinux_test.go

```diff
@@ -0,0 +1,63 @@
+//go:build selinux
+
+package selinux
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("Test:", func() {
+
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Nodes", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil,
+		)
+	})
+
+	It("Validate Pods", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus(),
+		)
+	})
+
+	It("Validate selinux is enabled", func() {
+		testcase.TestSelinuxEnabled()
```

Comment:
all tcs we need to make same signature to comply with model 

https://github.com/bguzman-3pillar/distros-test-framework/blob/f468580ee4c3b8f467bfeae8462fe1d39d1de0e3/pkg/customflag/model.go#L55-L56

Reply:
deleteWorkload bool